generator/uscore/uscore_generator.rb

```diff
@@ -131,6 +132,29 @@ def create_authorization_test(sequence)
         sequence[:tests] << authorization_test
       end
 
+      def create_revinclude_test(sequence)
+        revinclude_test = {
+          tests_that: "A Server SHALL be capable of supporting the following _revincludes: #{sequence[:revincludes].join(',')}",
```

Comment:
This is a different naming convention than we've been using so far.  The name doesn't have the "SHALL" requirement -- that information should be in the description of the test that you get when you click 'more' in the UI.

Reply:
Updated the description to be more like the other test descriptions.